cmd/load.go

```diff
@@ -238,7 +238,7 @@ func LoadCommand() *cobra.Command {
 	cmdLoadPackage.PersistentFlags().BoolVarP(&verbose, "verbose", "v", false, "verbose")
 	cmdLoadPackage.PersistentFlags().BoolVarP(&bridged, "bridged", "b", false, "bridge networking")
 	cmdLoadPackage.PersistentFlags().StringVarP(&imageName, "imagename", "i", "", "image name")
-	cmdLoadPackage.PersistentFlags().BoolVarP(&accel, "accel", "x", false, "use cpu virtualization extension")
+	cmdLoadPackage.PersistentFlags().BoolVarP(&accel, "accel", "x", true, "use cpu virtualization extension")
```

Comment:
As far as I can see, the shorthand version of a boolean flag with true as default value doesn't make much sense, because the shorthand version can only be used to set the flag to true. So I would drop the shorthand flag "x" and use BoolVar() instead.

Reply:
Agree, I've changed to BoolVar